scanner/src/main/kotlin/storages/PostgresStorage.kt

```diff
@@ -79,9 +79,12 @@ class PostgresStorage(
     }
 
     private fun createTable(): Boolean {
-        val query = """
-            CREATE SEQUENCE $schema.${table}_id_seq;
+        val statement = connection.createStatement()
```

Comment:
How about putting all three statements into a transaction?

Reply:
Will that tell me in some sort of response which command exactly failed, if any?

Reply:
I haven't checked as I just assumed that `statement.execute...()` would still throw a SQL exception if something failed. 

Does the SQLException actually also provide the desired information e.g. in its message or stack trace ?

Reply:
It should and it would rollback the changes if a command fails. But as no transaction was used before I think it's fine to keep it like that and safe the time, because when I migrate this to exposed everything will be done in transactions anyway.

Reply:
> Does the SQLException actually also provide the desired information e.g. in its message or stack trace ?

I can see from the mentioned source lines in the stack trace which `execute()` failed. But this is only valuable if I have one execute per command.

> But as no transaction was used before I think it's fine to keep it like that and safe the time, because when I migrate this to exposed everything will be done in transactions anyway.

:+1: